examples/all-clusters-app/all-clusters-common/src/rvc-modes.cpp

```diff
@@ -201,7 +200,6 @@ void emberAfRvcCleanModeClusterInitCallback(chip::EndpointId endpointId)
     VerifyOrDie(endpointId == 1); // this cluster is only enabled for endpoint 1.
```

Comment:
wait...what? Why is this only enabled on ep1?

Reply:
In the all-clusters-app it is only enabled on endpoint 1.